geoslurp/discover/oceanobs/argo.py

```diff
@@ -1,94 +0,0 @@
-# This file is part of geoslurp-tools.
```

Comment:
Same here: has this file been accidentally deleted? You may need to checkout the file from the most up to date master branch in your own branch first.